.buildkite/pipeline-utils/utils.test.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+
+import { expect } from 'chai';
+import { getKibanaDir, getVersionsFile } from './utils';
+import fs from 'fs';
+
+// TODO: replace mocha with jest, and write tests that mock FS
```

Comment:
Do you plan to add these tests in this PR or in the future?

Reply:
I'd prefer to keep it separate, not to expand the scope of this PR even further

Reply:
https://github.com/elastic/kibana/issues/187878 - what do you think?

Reply:
sgtm